packages/orama/src/id-shortener.ts

```diff
@@ -0,0 +1,54 @@
+import { IdStore } from './types.js'
+
+const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789!@#$%^&*()'
```

Comment:
We can use safely also:
- `=`
- `£` // Not sure
- `[`
- `]`
- `{`
- `}`
- `-`
- `+`
- `,`
- `.`
- `;`
- `:`
- `?`
- `|`
- `_`
- `~`

This reduces the string length.